foundry.toml

```diff
@@ -0,0 +1,21 @@
+[default]
+solc_version = "0.8.7"
+optimizer_runs = 1000000
+verbosity = 1
+fuzz_runs = 5
+remappings = [
+    'forge-std/=lib/forge-std/src/',
+    'ds-test/=lib/forge-std/lib/ds-test/src/',
+    'openzeppelin-contracts/=lib/openzeppelin-contracts/contracts/'
+]
+
+[ci]
+solc_version = "0.8.7"
+optimizer_runs = 1000000
+verbosity = 1
+fuzz_runs = 50000
+remappings = [
```

Comment:
In this template the remappings are in an individual file.

https://github.com/FrankieIsLost/forge-template

I don't have a strong opinion. Maybe we can just follow the standard. (If there is already a common approach how to do it. :) )

Reply:
That repo doesn't even have `foundry.toml` :thinking: I wouldn't be surprised if it used an old version of foundry, it evolves really fast!

Reply:
Ah, I think you might be right. 👍

Reply:
Right, the `foundry.toml` file was established recently.